package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "ast-types": "git+https://github.com/jlongster/ast-types.git",
     "babylon": "^6.15.0",
+    "esutils": "^2.0.2",
```

Comment:
Yeah, I wasn't sure, but IME tools like this end up including something like it or redoing it themselves.